app/controllers/webhooks_controller.rb

```diff
@@ -9,9 +9,11 @@ def email
         return
       end
 
-      if p.from.address == 'postmaster@hmps.gsi.gov.uk'
-        logger.error "Sender ( postmaster@hmps.gsi.gov.uk ) detected. Skipping message.\n #{p.inspect}"
-        render text: 'Sender postmaster@hmps.gsi.gov.uk'
+      blacklist = %w(postmaster sendmaster webmaster no-reply noreply root)
+
+      if blacklist.include? p.from.local
```

Comment:
We should lower-case the from address - technically PostMaster@a.com and postmaster@a.com are the same so lets just be safe about it.